README.md

```diff
@@ -2,15 +2,15 @@
 
 # Capstone One
 
-> Portfolio is one of the most powerful tools in  software developer toolbox; it’s the easiest way of showing what you’re truly capable of as a developer, and is a quick and simple way for recruiters and hiring managers to get an idea of what you can bring to their teams.
- This project will include my projects both personal and microverse assigned projects.
+> Project capstone 1: online website for a conference.
 
 ## Live preview
-
+My capstone project can be accessed  [here](https://isaacpitwa.github.io/conference_capstone/)
 
 ## Built With
 
-- HTML & CSS
+- HTML & CSS Sass
+- Javascript
 - GitFlow
 - Linters
 - Documented Work
```

Comment:
[OPTIONAL] It would be nicer to add some steps on how the user can use your project locally so your project is documented professionally.